Cython/Compiler/Code.py

```diff
@@ -1317,7 +1317,7 @@ def get_pyunicode_ptr_const(self, text):
             c = self.pyunicode_ptr_const_index[text] = self.new_const_cname()
         return c
 
-    def get_py_string_const(self, text, identifier=None,
+    def get_py_string_expr(self, text, identifier=None,
```

Comment:
Not easy to say what a good name is here. `expr` suggests a Python expression, i.e. an `ExprNode`.
`get_py_string_const_code()` ? It returns the C code to refer to a Python string constant.

Reply:
I guess it's `_cexpr` then instead of `_expr` then?